tests/platform_tests/test_auto_negotiation.py

```diff
@@ -56,10 +56,11 @@ def recover_ports(duthosts, enum_dut_portname_module_fixture, fanouthosts):
     logger.info('Collecting existing port configuration for DUT and fanout...')
     for duthost in duthosts:
         if dutname == 'unknown' or dutname == duthost.hostname:
-            all_ports = build_test_candidates(duthost, fanouthosts, portname)
```

Comment:
We will get a list that contains only 1 port by this way.
If we want to enumerate all ports, we have to put "all_ports" in it parameter list.